process/smartContract/changeOwnerAddress_test.go

```diff
@@ -0,0 +1,52 @@
+package smartContract
+
+import (
+	"testing"
+
+	"github.com/ElrondNetwork/elrond-go/data/state"
+	"github.com/ElrondNetwork/elrond-go/data/transaction"
+	"github.com/ElrondNetwork/elrond-go/process"
+	"github.com/ElrondNetwork/elrond-go/process/mock"
+	vmcommon "github.com/ElrondNetwork/elrond-vm-common"
+	"github.com/stretchr/testify/require"
+)
+
+func TestChangeOwnerAddress_ProcessBuiltinFunction(t *testing.T) {
+	t.Parallel()
+
+	coa := changeOwnerAddress{}
+
+	owner := []byte("sender")
+	tx := &transaction.Transaction{
+		SndAddr: owner,
+	}
+
+	addr := []byte("addr")
+	count := 0
```

Comment:
same as in my first comment. Use 2 different booleans instead of a common counter

Reply:
done.